src/functionalTest/java/uk/gov/hmcts/sptribs/testutil/TaskManagementService.java

```diff
@@ -42,8 +42,8 @@ public Response search(String caseId,
         taskMonitorService.triggerTerminationJob();
         taskMonitorService.triggerReconfigurationJob();
 
-        // Wait 90 seconds to allow time for task to be created in database
-        Thread.sleep(90000);
+        // Wait 70 seconds to allow time for task to be created in database
+        Thread.sleep(70000);
```

Comment:
Can you increase this to 80 seconds to be consistent with the other PRs please. We can decrease it later if needed. No need to run the WA functional tests again.